### PR TITLE
Add new 4x12 ortho layout & add credit to XD75 port

### PR DESCRIPTION
**Pull request by BenKesselring**
I've been doing a bit of playing around with a Planck/Let's Split keymap using Colemakl Mod-DH. I also added credit for the XD75 port because I noticed that "REPLACE_WITH_YOUR_NAME" bit at the top of every part of the xd75 port...

### CODE REVIEW

**Comment by jackhumbert** [comment]
Cool! Would you mind removing the image and linking to it in the readme? We're trying to reduce the size of the repo :)

**Comment by BenKesselring** [comment]
Sure thing! What image host is typically used for this?

**Comment by drashna** [comment]
Imgur is a real popular, and good one.

**Comment by BenKesselring** [comment]
Uploaded the image to imgur and removed it from the repo!

File: .gitignore [hunk 1]
```diff
@@ -44,9 +44,9 @@ util/Win_Check_Output.txt
 .stfolder
 
 # ignore image files
-*.png
-*.jpg
-*.gif
+.png
+.jpg
+.gif
```
Comment (jackhumbert) [inline .gitignore]: I think these need to stay `*.` - we don't want any images in the repo :)

**Comment by BenKesselring** [comment]
Oh crap! I spaced on fixing the gitignore. I'll fix that when I get home.

**Comment by jackhumbert** [comment]
Perfect - thanks :)

**Pull request merged**